.github/workflows/main.yml

```diff
@@ -0,0 +1,69 @@
+name: Serverless Plugin For Each CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  Setup-Env:
+     runs-on: ubuntu-latest
+     container: bash
+     timeout-minutes: 1
+     outputs:
+       isMain: ${{ steps.setupEnv.outputs.isMain }}
+     steps:
+       - name: Setup Env
+         id: setupEnv
+         uses: Brightspace/actions-parse-workflow-env@v2
+
+  Tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    strategy:
+      matrix:
+        node: [10, 12, 14]
+    steps:
+      - name: Checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+
+      - name: Setup Node
+        uses: Brightspace/third-party-actions@actions/setup-node
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Download Dependencies
+        run: npm ci
+
+      - name: Unit Tests
+        run: npm run test:unit
+
+      - name: Lint
+        run: npm run lint
+
+      - name: License Test
+        run: npm run license
+
+  Release:
+    if: ${{ needs.Setup-Env.outputs.isMain == 'true' }}
+    runs-on: ubuntu-latest
+    needs: [Setup-Env, Tests]
+    timeout-minutes: 5
+    steps:
+      - name: Checkout
+        uses: Brightspace/third-party-actions@actions/checkout
+        with:
+          persist-credentials: false
+
+      - name: Setup Node
+        uses: Brightspace/third-party-actions@actions/setup-node
+        with:
+          node-version: 12
+
+      - name: Semantic Release
+        uses: BrightspaceUI/actions/semantic-release@master
```

Comment:
Guessing the default for this is npmjs.or and not artifactory.

Reply:
this is a public repo, so we should use NPM

Reply:
Yup, just making sure our homebrew action defaults to npmjs.org